mod_reforged/hooks/entity/tactical/enemies/skeleton_priest.nut

```diff
@@ -26,7 +26,7 @@
 		if (::Reforged.Config.IsLegendaryDifficulty)
     	{
     		this.m.Skills.add(::new("scripts/skills/perks/perk_anticipation"));
-    		this.m.Skills.add(::new("scripts/skills/perks/perk_nimble"));
+    		this.m.Skills.add(::new("scripts/skills/effects/rf_soul_link_effect"));
```

Comment:
I don't find it very thematic for skeleton priests to have this skill. For now, let's add it only on Hexe and Necromancer.